home01/js/script_10.js

```diff
@@ -0,0 +1,11 @@
+// 10. Пользователь вводит сумму вклада в банк на 2 месяца, с процентной ставкой депозита 5% годовых. Вывести сумму начисленных процентов.
+const summAmount = +prompt(`Enter deposit amount:`);
+const Rate = +prompt(`Enter deposit rate:`);
```

Comment:
используем `camelCase`
```suggestion
const rate = +prompt(`Enter deposit rate:`);
```